packages/now-next/src/index.ts

```diff
@@ -303,7 +303,8 @@ export const build = async ({
         entryDirectory,
         pathsInside,
         files,
-        urls[entrypoint]
+        urls[entrypoint],
+        meta.isDev
```

Comment:
```suggestion
        urls[entrypoint]
```
This method is only used in dev mode so we don't need to pass whether we're in dev mode or not

Reply:
@ijjk I think its better to be explicit, someone might move this getRoutes() to a different place where it might get used in production.

Reply:
hmm i reverted this to resolve this comment but I can re-revert :)